ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/RecordFormatter.java

```diff
@@ -109,11 +109,15 @@ public RecordFormatter(
 
     final List<String> formatted = formatRecords(records);
 
-    final boolean sameKeyFormatChanged = activeKeyFormat
-        .equals(keyDeserializers.getPossibleFormats().get(0));
-
-    final boolean sameValueFormatChanged = activeValueFormat
-        .equals(valueDeserializers.getPossibleFormats().get(0));
+    final boolean sameKeyFormatChanged = keyDeserializers.getPossibleFormats().stream()
+        .map(activeKeyFormat::equals)
+        .findFirst()
+        .orElse(false);
```

Comment:
nit, this and the one below can be replaced with this pattern:
```java
keyDeserializers
  .getPossibleFormats()
  .stream()
  .anyMatch(activeKeyFormat::equals);
```

Reply:
Done!

Reply:
Thanks again! I'll wait for the build to be green and merge.